dist/self-service-reasons/1.0.5/cancellation-reasons_ro.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "formular anulare",
```

Comment:
does this matter? it stayed "cancelForm" for the other languages @camilavcoutinho